README.md

```diff
@@ -258,7 +280,7 @@ The above will call the class's redirect method, if it can't find a user in Lara
 
 ## Contributing
 
-Thank you for considering contributing to the Laravel Azure Middleware! To encourage active collaboration, we encourage pull requests, not just issues.
+Thank you for considering contributing to the Laravel Azure Middleware! To encourage active collaboration, the project encourages you to make pull requests, not just issues.
```

Comment:
I'm okay keeping this `we encourage` since, we, Root, encourage it.

Reply:
Fixed in latest push